tools/Evm/Evm/Evm.csproj

```diff
@@ -5,6 +5,7 @@
         <TargetFramework>net9.0</TargetFramework>
         <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
+        <Version>1.0.0.0</Version>
```

Comment:
Are 4 parts needed? We normally go with semantic versioning which has major.minor.patch